scripts/tests/test_validate_docstrings.py

```diff
@@ -907,12 +921,14 @@ def test_exit_status_for_validate_one(self, monkeypatch):
         exit_status = validate_docstrings.main(func_name='docstring1',
                                                prefix=None,
                                                errors=[],
-                                               output_format='default')
+                                               output_format='default',
+                                               ignore_deprecated=False)
         assert exit_status == 0
 
     def test_exit_status_errors_for_validate_all(self, monkeypatch):
         monkeypatch.setattr(
-            validate_docstrings, 'validate_all', lambda prefix: {
+            validate_docstrings, 'validate_all',
+            lambda prefix, ignore_deprecated=None: {
```

Comment:
Sorry I didn't see that before... `ignore_deprecated` expects a boolean value, I don't think we should pass None. And I see that it has a default value of `False` that is what we want. So, can you restore the original code? I think it should work, right?

Reply:
ahh, i did this because i thought right now, `validate_all` takes in two arguments, and although `ignore_deprecated` has beed set to False by default, when we use lambda to create this mock data, we still need to assign a variable as a kind of placeholder, otherwise, there will be an error raised... can I still keep this, and remove None so that it looks like `lambda: prefix, ignore_deprecated` , please share your thoughts! And thanks for the quick response!

Reply:
oh, that's right, I checked it fast and didn't see those are the arguments of the mock function, not the parameters being passed. In that case set `ignore_deprecated=False`, so we have the same signature in the mock as in the mocked function.

Reply:
oh, yeah, changed to False! thx for the feedback!